new-integration-tests/src/test/java/oracle/weblogic/kubernetes/ItUsabilityOperatorHelmChart.java

```diff
@@ -199,49 +258,451 @@ public void testDeleteOperatorButNotDomain() {
+      assertNull(opHelmParam2, "FAILURE: Helm installs operator in the same namespace as first operator installed ");
+    } finally {
+      deleteSecret("ocir-secret",opNamespace);
+      cleanUpSA(opNamespace);
+      cleanUpSA(op2Namespace);
+      uninstallOperator(opHelmParams);
+      uninstallOperator(op2HelmParams);
+    }
+  }
+
+  /**
+   * Intitialize two operators op1 and op2 with same ExternalRestHttpPort.
+   * Install operator op1.
+   * Install operator op2.
+   * Installation of second operator should fail.
+   *
+   * @throws Exception when second operator installation does not fail
+   */
+  @Test
+  @DisplayName("Negative test to try to create the operator with not preexisted namespace")
+  @Slow
+  public void testSecondOpSharingSameExternalRestPortNegativeInstall() {
+    String opServiceAccount = opNamespace + "-sa";
+    String op2ServiceAccount = op2Namespace + "-sa2";
+    String opReleaseName = OPERATOR_RELEASE_NAME + "1";
+    HelmParams op1HelmParams = new HelmParams().releaseName(opReleaseName)
+        .namespace(opNamespace)
+        .chartDir(OPERATOR_CHART_DIR);
+    HelmParams opHelmParams = installOperatorHelmChart(opNamespace, opServiceAccount, true, true,
+        true,null,"deployed", 0, op1HelmParams, false, domain1Namespace);
+    int externalRestHttpsPort = getServiceNodePort(opNamespace, "external-weblogic-operator-svc");
+    String op2ReleaseName = OPERATOR_RELEASE_NAME + "2";
+    HelmParams op2HelmParams = new HelmParams().releaseName(op2ReleaseName)
+        .namespace(op2Namespace)
+        .chartDir(OPERATOR_CHART_DIR);
+
+    // install and verify operator2
+    logger.info("Installing and verifying operator2 fails");
+    try {
+      String expectedError = "Error: Service \"external-weblogic-operator-svc\" "
+          + "is invalid: spec.ports[0].nodePort: Invalid value";
+      HelmParams opHelmParam2 = installOperatorHelmChart(op2Namespace, op2ServiceAccount,
+          true, true, true,
+          expectedError,"failed",
+          externalRestHttpsPort, op2HelmParams, false, domain2Namespace);
+      assertNull(opHelmParam2, "FAILURE: Helm installs operator in the same namespace as first operator installed ");
+    } finally {
+      cleanUpSA(opNamespace);
+      cleanUpSA(op2Namespace);
+      deleteSecret("ocir-secret",opNamespace);
+      deleteSecret("ocir-secret",op2Namespace);
+      uninstallOperator(opHelmParams);
+      uninstallOperator(op2HelmParams);
+    }
+  }
+
+  /**
+   * Install the operator with non existing operator namespace.
+   * The helm install command should fail.
+   *
+   * @throws Exception when helm install does not fail
+   */
+  @Test
+  @DisplayName("Negative test to try to create the operator with not preexisted namespace")
+  @Slow
+  public void testNotPreCreatedOpNsCreateOperatorNegativeInstall() {
+    // install and verify operator
+    logger.info("Installing and verifying operator");
```

Comment:
logger.info("Installing and verifying operator"); is being repeated few lines below for different set of code/action.

Reply:
cleaned